openfl/component/aggregation_functions/yogi_adaptive_aggregation.py

```diff
@@ -26,7 +26,7 @@ def __init__(
         *,
         agg_func: AggregationFunction = DEFAULT_AGG_FUNC,
         params: Optional[Dict[str, np.ndarray]] = None,
-        model_interface=None,
+        model_interface: 'ModelInterface' = None, # NOQA
```

Comment:
```suggestion
        model_interface: ModelInterface = None,
```
If here the problem with postponed evaluation please use `from __future__ import annotations`